src/pybind/mgr/prometheus/module.py

```diff
@@ -108,6 +108,13 @@ def health_status_to_number(status):
 
 NUM_OBJECTS = ['degraded', 'misplaced', 'unfound']
 
+HEALTH_CHECKS = [
```

Comment:
nit, might want to use `namedtuple` here.

Reply:
ACK - great idea.